lib/frame.js

```diff
@@ -0,0 +1,30 @@
+class Frame {
+  constructor () {
+    this.rolls = []
+    this.score = 0
+  }
+
+  feedRoll(number) {
+    this.rolls.push(number);
+  }
+
+  calculateScore() {
+
+    if (this.rolls.length < 2) {
+      this.score = 0;
+    } else if (this.rolls.length === 2) {
+      if (this.rolls[0] + this.rolls[1]  < 10) {
+        this.score = this.rolls[0] + this.rolls[1];
+      } else if (this.rolls[0] + this.rolls[1]  === 10) {
```

Comment:
In line 18 does this represent a spare? If so should line 19 be written as:
``this.score = this.rolls[0] + this.rolls[1];``